### ISSUE TITLE
data/chinese_word2vec has bugs

### ISSUE THREAD

**Issue by wenh06** [thunlp/OpenAttack#218]
the `LOAD` method in [data/chinese_word2vec](https://github.com/thunlp/OpenAttack/blob/master/OpenAttack/data/chinese_word2vec.py) has bugs. You can try with
```python
from collections import Counter
Counter(word2id.values()).most_common()
```
and see what will happen. The point is that the `word`s have duplicates.
The correct ways should be
```python
# 2x faster, but much more memory consuming
content = list(filter(lambda l: len(l)==301, [line.strip().split(' ') for line in f.readlines()]))
id2vec = np.array([line[1:] for line in content], dtype=float)
word2id = {line[0]:i for i,line in enumerate(content)}
```
or
```python
id2vec = []
word2id = {}
for idx, line in enumerate(f.readlines()):
    tmp = line.strip().split(' ')
    word = tmp[0]
    embed = np.array([float(x) for x in tmp[1:]])
    if len(embed) != 300:
        continue
    word2id[word] = idx
    id2vec.append(embed)
    print(f"processed {idx+1} lines", end="\r")
id2vec = np.stack(id2vec)
```

**Comment by wenh06** [comment]
Similar problem in [counter_fit](https://github.com/thunlp/OpenAttack/blob/master/OpenAttack/data/counter_fit.py), but I haven’t checked if the words in it have duplicates or not.

The problem is that the indices are shifted immediately after the first occurrence of the duplicates.

**Comment by a710128** [comment]
Thank you for your feedback, we will fix this issue in the next release.

**Comment by cgq15** [comment]
Fixed in #237
